Python/crossfire/crossfire/occurrences.py

```diff
@@ -1,18 +1,39 @@
 from queue import Empty, Queue
 from urllib.parse import urlencode
 
+from crossfire.errors import CrossfireError
+
+TYPE_OCCURRENCES = {"all", "withVictim", "withoutVictim"}
+
+
+class UnknownTypeOccurrenceError(CrossfireError):
+    def __init__(self, type_occurrence):
+        message = ("""Unknown type_occurrence""",)
+        f"`{type_occurrence}`. Valid formats are: {', '.join(TYPE_OCCURRENCES)}"
+        super().__init__(message)
+
 
 class Occurrences:
-    def __init__(self, client, id_state, id_cities=None, limit=None, format=None):
+    def __init__(
+        self,
+        client,
+        id_state,
+        id_cities=None,
+        limit=None,
+        format=None,
+        type_occurrence="all",
+    ):
         self.client = client
         self.format = format
         self.limit = limit
+        if type_occurrence and type_occurrence not in TYPE_OCCURRENCES:
```

Comment:
Pq checar `if type_occurrence ` primeiro?

Se `type_occurence` for qualquer coisa que não os valores de `TYPE_OCCURRENCES`, vai falhar… isso inclui `None`, `False`, `0`, `""`, `[]` etc.

Reply:
Eu fiz o commit e pensei sobre isso.
Dev ter corrigido para não te dar trabalho.
No proximo commit estará corrigido.

Reply:
Não está resolvido ainda. Ainda continuo vendo o `if` desnecessário. Não é boa prática marcar como “resolvido” _antes_ de “resolver”

Acertando isso, está 💯

Reply:
"Mala mía", de novo. Foi corrigido no último commit.